params/config_test.go

```diff
@@ -125,7 +125,7 @@ func TestCheckCompatible(t *testing.T) {
 			},
 		},
 		{
-			stored:        TestChainConfig,
+			stored:        TestApricotPhase5Config,
```

Comment:
This gives a different error because now `TestChainConfig` activates `Shanghai` and `TestApricotPhase4Config` not; and since we first check the eth upgrade errors it returns the `Shanghai` error rather than `AP5` error. If we want we can just change the expected error here as well.